spec/features/spree/admin/taxjar_settings_spec.rb

```diff
@@ -23,6 +24,13 @@
       click_on "TaxJar Settings"
     end
 
+    it "the user backfills their transactions" do
```

Comment:
With capybara `scenario` is an alias of `it` that makes these kinds of tests read better.

Reply:
Fixed!